keps/sig-node/4191-split-image-filesystem/kep.yaml

```diff
@@ -0,0 +1,41 @@
+title: Splitting the Image Filesystem
+kep-number: 4191
+authors:
+  - "@kannon92"
+owning-sig: sig-node
+participating-sigs:
+  - sig-storage
+status: implementable
+creation-date: 2023-09-06
+reviewers:
+  - "@rphillips"
+  - "@ruiwen-zhao"
+  - "@haircommander"
+approvers:
+  - "@dchen1107"
+
+see-also:
+  - "/keps/sig-node/2371-cri-pod-container-stats"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.29"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.29"
+
+# The following PRR answers are required at alpha release
+# List the feature gate name and the components for which it must be enabled
+feature-gates:
+  - name: KubeletSeparateDiskGC
+    components:
+      - kubelet
+disable-supported: false
+
+# The following PRR answers are required at beta release
+metrics:
```

Comment:
you mentioned a metric above - you should add it here too

Reply:
I thought this was about new metrics you add. Evictions is an existing one.

Reply:
I added `evictions_total` to this.

Reply:
Turns out node_controller_evictions_total is the name.